addon/components/uni-mobile-number.js

```diff
@@ -2,11 +2,13 @@ import Ember from 'ember';
 import { getAllCountryNames, getCountryName, getCountryCallingCode } from 'ember-cli-countries/utils/countries';
 import layout from '../templates/components/uni-mobile-number';
 import KeyCodes from 'ember-cli-uniq/enums/key-codes-type';
+import CountryCodes from 'ember-cli-uniq/enums/country-codes-type';
 
-const { Component, isPresent } = Ember;
+const { Component, isPresent, computed } = Ember;
 
 const SEPARATOR_KEY = '__SEPARATOR__';
 const SEPARATOR_VALUE = '--------------------------------';
+const PT_PHONE_REGEXP = /^(?:[92]\d{2}(?:\s?\d{3}){2})$/;
```

Comment:
Will we be adding validations for phone numbers of other countries? If we should extract this to a util and have the regular expressions on an enum.

Reply:
for now, no, but I'll extract this to an util